PatientMonitoringSystem/PatientMonitoringSystem/Database/DatabaseHandler.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Data;
+using System.Data.SqlClient;
+using System.Configuration;
+
+namespace PatientMonitoringSystem.Database
+{
+	public class DatabaseHandler
+	{
+
+		private static DatabaseHandler db_instance = null;
+		private static SqlConnection connection;
+		private static readonly object _lock = new object();
+		private static String connection_str;
+		private DatabaseHandler()
+		{
+			string connection_str = @"Data Source=(LocalDB)\MSSQLLocalDB;AttachDbFilename=|DataDirectory|\Database\Database1.mdf;Integrated Security=True";
+			connection = new SqlConnection(connection_str);
+		}
+
+		public static DatabaseHandler Instance
+		{
+			get
+			{
+				lock (_lock)
+				{
+					if (db_instance == null)
+					{
+						db_instance = new DatabaseHandler();
+					}
+				}
+
+				return db_instance;
+			}
+		}
+
+		public static void ExecuteStatement(SqlCommand command)
```

Comment:
@harrytpstudent Normally, with the singleton pattern, the only two class members which are marked as `static` are the factory method or property, e.g. `public static void GetInstance()` or `public static Instance`, and the backing field, e.g. `private static MyType instance`. If you remove the unnecessary `static` keywords in this class, you should be able to execute statements like `DatabaseHandler.Instance.ExecuteStatement(command)` instead of `DatabaseHandler.ExecuteStatement(command)`.